### DISCUSSION TITLE
Contact to Account conversion or to a Lead ?

### DISCUSSION THREAD

**Discussion by testmodem** [YetiForceCompany/YetiForceCRM#14411] [General]
Dear all

Is it possible to convert a contact to an Account ? 

Thanks

**Comment by mariuszkrzaczkowski** [comment]
https://github.com/YetiForceCompany/YetiForceCRM/issues/14361
https://github.com/YetiForceCompany/YetiForceCRM/issues/9763

**Reply by testmodem** [reply]
Thanks for your prompt response. I followed both the links and understand that they are related to conversion of **Lead-to-Account**. Is the same possible for **Contact-to-Account** Conversion ?

![Contact2Leads](https://user-images.githubusercontent.com/59494950/111473473-e427a600-874c-11eb-84d5-ec3abfcb0cdb.png)

**Reply by mariuszkrzaczkowski** [reply]
no, see the mapping maybe it will help https://gitdeveloper.yetiforce.com/index.php?module=MappedFields&parent=Settings&view=List

**Reply by testmodem** [reply]
Dear mariuszkrzaczkowski

Thanks for your support.
Its working .

**Reply by testmodem** [reply]
Dear mariuszkrzaczkowski 
Can i map "Account name" in Account module to "Lastname" in Contact module ?

![account to contact](https://user-images.githubusercontent.com/59494950/111913338-80281900-8a8f-11eb-9731-d3cd408035b8.png)


Thanks

**Reply by vovpff** [reply]
This should work because this two fields is text type.

**Reply by testmodem** [reply]
> This should work because this two fields is text type.

That is the problem, Its not working (: .. When I select "Last name" in Contacts for mapping , "Account name" disappears from "Account" dropdown .

![Stage 1 Mapping](https://user-images.githubusercontent.com/59494950/111941613-396c0a80-8af3-11eb-9667-0e65b735ee55.png)
![Stage 2A Selecting Lastname in Contacts](https://user-images.githubusercontent.com/59494950/111941632-44269f80-8af3-11eb-965a-915ad1bf416d.png)
![Stage 2B Account Name dissappear after Lastname selecttion](https://user-images.githubusercontent.com/59494950/111941637-4557cc80-8af3-11eb-9452-b22854293b3e.png)

**As long I dont select anything in under "Contact", "Account name" remains available for selection.**

![Before Selectings Lastname in Contact - Accountname available](https://user-images.githubusercontent.com/59494950/111942109-7258af00-8af4-11eb-984c-a1425e933e9f.png)

**Reply by testmodem** [reply]
Any suggestions ?

**Reply by vovpff** [reply]
Idea. I think this is because Account last name and Account organisation name this is the same field with different visible name (made for private individual companies in 6.1).
